lib/rummager_finder_publisher.rb

```diff
@@ -9,8 +9,14 @@ def initialize(metadatas, logger: Logger.new(STDOUT))
 
   def call
     metadatas.each do |metadata|
-      if metadata[:file].has_key?("content_id")
+      if metadata[:file].has_key?("content_id") && !preview_only?(metadata)
```

Comment:
We should remove the `content_id` check entirely and only use the feature flag - if a finder is merged then it should be fine for it to be published on Preview, and having two mechanisms for preventing publishing is confusing to everyone and complicates the logic here. Since `PublishingApiFinderPublisher` also checks the `content_id` I'd be happy for that to be a separate PR removing that check from both classes, rather than tying that change in behaviour to this fix.